src/openrct2/world/MapGen.cpp

```diff
@@ -259,9 +258,9 @@ static void mapgen_place_tree(sint32 type, sint32 x, sint32 y)
 static void mapgen_place_trees()
 {
     sint32 numGrassTreeIds = 0, numDesertTreeIds = 0, numSnowTreeIds = 0;
```

Comment:
You can remove these variables now.

Reply:
They're currently used to insert new elements. How should I insert new elements instead?

Reply:
`snowTreeIds.size()`